oneview/resource_appliance_ssh_access.go

```diff
@@ -0,0 +1,96 @@
+// (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+package oneview
+
+import (
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceSSHAccess() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceSSHAccessRead,
+		Update: resourceSSHAccessUpdate,
+		Delete: resourceSSHAccessDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"allow_ssh_access": {
+				Type:     schema.TypeBool,
+				Default:  false,
+				Optional: true,
```

Comment:
This one is a required field right...?

Reply:
we cannot mark it as required when we provide the default value